test/verify/check-networkmanager-wireguard

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/python3 -cimport os, sys; os.execv(os.path.dirname(sys.argv[1]) + "/../common/pywrap", sys.argv)
+
+from testlib import MachineCase, test_main
```

Comment:
See commit a703c070bb49394, please change this to `import testlib` for being consistent with other tests.